packages/plugins/sentry/server/src/config.ts

```diff
@@ -0,0 +1,16 @@
+import type { NodeOptions } from '@sentry/node';
+
+export interface Config {
+  dsn: string | null;
+  sendMetadata: boolean;
+  init: NodeOptions;
+}
+
+export default {
+  default: {
+    dsn: null,
+    sendMetadata: true,
+    init: {},
+  } as Config,
```

Comment:
i think `satisfies` is better here because you're checking as opposed to enforcing?